cli/providers/aws/route53/cnames.go

```diff
@@ -0,0 +1,127 @@
+package route53
+
+import (
+	"fmt"
+
+	awsSDK "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/route53"
+	"github.com/pchico83/i2kit/cli/providers/aws"
+	"github.com/pchico83/i2kit/cli/providers/aws/cf"
+	"github.com/pchico83/i2kit/cli/schemas/environment"
+	"github.com/pchico83/i2kit/cli/schemas/service"
+)
+
+//Create creates a CNAME resolving to a service
+func Create(s *service.Service, e *environment.Environment) error {
+	config, err := aws.GetConfig(e.Provider)
+	if err != nil {
+		return err
+	}
+	stack, err := cf.Get(s.Name, config)
+	if err != nil {
+		return err
+	}
+	config, err = aws.GetConfig(e.Root)
+	if err != nil {
+		return err
+	}
+	for _, o := range stack.Outputs {
+		if *o.OutputKey == "elbURL" {
+			svc := route53.New(session.New(), config)
+			hostedZonesInput := &route53.ListHostedZonesByNameInput{
+				DNSName:  awsSDK.String(e.Root.HostedZone),
+				MaxItems: awsSDK.String("1"),
+			}
+			resp, err := svc.ListHostedZonesByName(hostedZonesInput)
+			if err != nil {
+				return err
+			}
+			if len(resp.HostedZones) != 1 {
```

Comment:
could there be 2 hosted zones if we have a public and a private?

Reply:
Not yet